### ISSUE TITLE
Login always says "This email address is invalid"

### ISSUE THREAD

**Issue by 1f5** [chrisstaite/ZEServices#2]
When trying to log in, after pushing the "Sign in" Button, the app always says "This email address is invalid", no matter what e-mail address ist used. I can use the same credentials to successfully log into Zeddy and "My Renault" app.
It's weird, because the isEmailValid() function in LoginActivity.java at Line 260 looks fine.

**Comment by chrisstaite** [comment]
Which country is your account from?

**Comment by 1f5** [comment]
Germany. The email provider is gmx.de
However, when trying to input other email addresses, that do not have an account, i would expect to see the "No accounts for the login"-message instead of "This email address is invalid".

**Comment by 1f5** [comment]
I figured it out. Sorry for bothering you. The issue was caused by Energized Protections filter list blocking socialize.eu1.gigya.com and accounts.eu1.gigya.com. I already filed an issue to resolve this.
However, i found the error message in this case misleading. I would rather have expected the "No accounts for the login"-Message or maybe a specific message for this type of error.

Thanks for your fast reply and for building this great piece of software. Keep up the good work!
